index.html

```diff
@@ -5450,7 +5450,21 @@ <h2>Acknowledgments</h2>
     : "undecided";
   if (conclusionNode) processTag(conclusionNode, conclusionDict);
 
-  // TODO: extract the “supported use cases” & generate cross references
+  // extract the “supported use cases” & generate cross references
+  const capabilityUseCasesList = node.querySelector("ul[data-ucr-role='capability-uses']");
+  if (capabilityUseCasesList) {
+    const supportedUseCaseLinks = [...document.querySelectorAll(`[data-ucr-role='use-case'] [href='#${id}']`)];
+    supportedUseCaseLinks.forEach((link) => {
+      const useCase = link.closest("[data-ucr-role='use-case']");
+      const useCaseTitle = findSectionTitle(useCase);
```

Comment:
ReSpec will insert the title and section number for an empty `<a></a>` that links to another section in the document, so we can skip this.

```suggestion
```